js/react_native/ios/OnnxruntimeModule.mm

```diff
@@ -22,7 +22,7 @@
 #include "onnxruntime_extensions.h"
 #else
 #include "onnxruntime/coreml_provider_factory.h"
-#include "onnxruntime/onnxruntime_cxx_api.h"
+#import <onnxruntime/onnxruntime_cxx_api.h>
```

Comment:
why do we use `#import` here? it's not an Objective-C header.

Reply:
yea, so I am not 100% sure yet how the ios project header file path all wired up, but from the code history, we used to have this https://github.com/microsoft/onnxruntime/pull/15800/files#diff-6cd96aa3c0515756787745b690d4b9fae90a00e27bbdcc289f4b825f11979f3dL9 syntax for including headers in the js/react_native/ios which works with xcodebuild.

Reply:
tried on another branch `#include` with angle brackets would work too. will update.